prebid-server/developers/module-atags.md

```diff
@@ -227,6 +229,117 @@ of the actual object will differ in PBS-Java and PBS-Go.
+
+- Server-side analytics are the only game in town when it comes to App, AMP, DOOH, etc.
+- However, when Prebid.js is in use and bidders are split between client-side and server-side, it would be far better if auctions were only logged once.
+
+This feature allows all relevant data passed to the client from Prebid Server so that client-side analytics can be the one to log the results.
+To allow the sharing of these details, there are two conditions:
+
+1. Server-side account configuration must allow sharing of these details by setting `analytics.allow-client-details: true`
+1. The ORTB request must contain `ext.prebid.analytics.options.enableclientdetails: true`
+
+If both are true, then any and all PBS analytics tags will be copied to the response field ext.prebid.analytics.tags.
+
+### Client aTag example
+
+The "pb-ortb-blocking" module at the "processed auction" stage adds the following Analytics Tags (from the ORTB2 blocking module)
+
+```json
+[{
+   activities: [{
+    name: "enforce-blocking",
+    status: "success",
+    results: [{
+        status: "success-block",
+        values: {
+                  "attributes": ["badv"],
+                  "adomain": ["bad.com"]
+        },
+        appliedto: {
+          "bidder": "bidderA",
+           impids: ["1"]
+        }
+    },{
+        status: "success-allow",
+        appliedto: {
+          "bidder": "bidderA",
+          "impids": ["2","3","4"]
+        }
+    }]
+ }]
+```
+
+Also, the "vendorA-brand-safety" module at the "all processed bid responses" stage adds these aTags:
+
+```json
+[{
+   activities: [{
+    name: "brand-safety",
+    status: "success",
+    results: [{
+        status: "success-allow",
+        appliedto: {
+          "bidder": "bidderA",
+          "impids": ["1,","2","3","4"]
+        }
+    }]
+ }]
+```
+
+The resulting response with the request ext.prebid.analytics.options.enableclientdetails: true and config analytics.options.enableclientdetails:true would be:
+
+```json
+ext.prebid.analytics.tags: [{
+  "stage": "processed-auction-request",
+  "module": "pb-ortb-blocking",
+  "analyticstags": [{
+     activities: [{
+      name: "enforce-blocking",
+      status: "success",
+      results: [{
+        status: "success-block",
+        values: {
+                  "attributes": ["badv"],
+                  "adomain": ["bad.com"]
+        },
+        appliedto: {
+          "bidder": "bidderA",
+           impids: ["1"]
+        }
+    },{
+        status: "success-allow",
+        appliedto: {
+          "bidder": "bidderA",
+          "impids": ["2","3","4"]
+        }
+    }]
+ }]
+},{
+  "stage": "all-processed-bid-responses",
+  "module": "vendorA-brand-safety",
+  "analyticstags": [{
+   activities: [{
+    name: "brand-safety",
+    status: "success",
+    results: [{
+        status: "success-allow",
+        appliedto: {
+          "bidder": "bidderA",
+          "impids": ["1,","2","3","4"]
+        }
+    }]
+}]
```

Comment:
```suggestion
```json5
// this is actually a nested object - but aggregated for readablility
"ext.prebid.analytics.tags": [{
  "stage": "processed-auction-request",
  "module": "pb-ortb-blocking",
  "analyticstags": [{
     "activities": [{
      "name": "enforce-blocking",
      "status": "success",
      "results": [{
        "status": "success-block",
        "values": {
                  "attributes": ["badv"],
                  "adomain": ["bad.com"]
        },
        "appliedto": {
          "bidder": "bidderA",
           "impids": ["1"]
        }
    },{
        "status": "success-allow",
        "appliedto": {
          "bidder": "bidderA",
          "impids": ["2","3","4"]
        }
    }]
 }]
},{
  "stage": "all-processed-bid-responses",
  "module": "vendorA-brand-safety",
  "analyticstags": [{
   "activities": [{
    "name": "brand-safety",
    "status": "success",
    "results": [{
        "status": "success-allow",
        "appliedto": {
          "bidder": "bidderA",
          "impids": ["1,","2","3","4"]
        }
    }]
}]
```